burrito_link/burrito_link.c

```diff
@@ -260,13 +286,8 @@ int connect_and_or_send() {
         memcpy(SendBuf + BufLength, &lc->uiState, sizeof(lc->uiState));
         BufLength += sizeof(lc->uiState);
 
-        // UINT32
-
-        // printf("map_offset_x: %f\n", lc->playerX - lc->mapCenterX);
-        // printf("map_offset_y: %f\n", lc->playerY - lc->mapCenterY);
-        // printf("mapScale: %f\n", lc->mapScale);
-        // printf("compassRotation: %f\n", lc->compassRotation); // radians
-        // printf("UI State: %i\n", lc->uiState); // Bitmask: Bit 1 = IsMapOpen, Bit 2 = IsCompassTopRight, Bit 3 = DoesCompassHaveRotationEnabled, Bit 4 = Game has focus, Bit 5 = Is in Competitive game mode, Bit 6 = Textbox has focus, Bit 7 = Is in Combat
+        memcpy(SendBuf + BufLength, &lc->mountIndex, sizeof(lc->mountIndex));
```

Comment:
These lines don't match the stated goal of the pull request. It seems to be new information sent to the data stream. The code receiving this data stream should be edited to reflect this.